cefpython/browser.pyx

```diff
@@ -739,9 +739,9 @@ cdef class PyBrowser:
                         bool(pyEvent["focus_on_editable_field"])
             self.GetCefBrowserHost().get().SendKeyEvent(cefEvent)
 
-        cpdef py_void SendMouseClickEvent(self, x, y,
+        cpdef py_void SendMouseClickEvent(self, int x, int y,
```

Comment:
There is this comment in SendMouseClickEvent. Are these flags exported? If so this comment should be removed. If not we should probably expose these.

            """
            TODO: allow to pass modifiers which represents
                  bit flags describing any pressed modifier keys.
                  See cef_event_flags_t for values.
            enum cef_event_flags_t {
                EVENTFLAG_NONE                = 0,
                EVENTFLAG_CAPS_LOCK_ON        = 1 << 0,
                EVENTFLAG_SHIFT_DOWN          = 1 << 1,
                EVENTFLAG_CONTROL_DOWN        = 1 << 2,
                EVENTFLAG_ALT_DOWN            = 1 << 3,
                EVENTFLAG_LEFT_MOUSE_BUTTON   = 1 << 4,
                EVENTFLAG_MIDDLE_MOUSE_BUTTON = 1 << 5,
                EVENTFLAG_RIGHT_MOUSE_BUTTON  = 1 << 6,
                // Mac OS-X command key.
                EVENTFLAG_COMMAND_DOWN        = 1 << 7,
                EVENTFLAG_NUM_LOCK_ON         = 1 << 8,
                EVENTFLAG_IS_KEY_PAD          = 1 << 9,
                EVENTFLAG_IS_LEFT             = 1 << 10,
                EVENTFLAG_IS_RIGHT            = 1 << 11,
            """